src/components/Post/PostCommentForm.tsx

```diff
@@ -9,27 +9,42 @@ import React, { FC, useEffect, useState } from 'react';
 import ErrorAlert from 'src/ui-components/ErrorAlert';
 import UserAvatar from 'src/ui-components/UserAvatar';
 import styled from 'styled-components';
-
 import { ChangeResponseType } from '~src/auth/types';
 import { usePostDataContext, useUserDetailsContext } from '~src/context';
 import CommentSentimentModal from '~src/ui-components/CommentSentimentModal';
 import nextApiClientFetch from '~src/util/nextApiClientFetch';
-
 import ContentForm from '../ContentForm';
 import queueNotification from '~src/ui-components/QueueNotification';
 import { NotificationStatus } from '~src/types';
+import { Input } from 'antd';
 import { IComment } from './Comment/Comment';
 import { getSubsquidLikeProposalType } from '~src/global/proposalType';
+import SadDizzyIcon from '~assets/icons/sentiments-icons/sad-dizzy.svg';
+import SadIcon from '~assets/icons/sentiments-icons/sad.svg';
+import NeutralIcon from '~assets/icons/sentiments-icons/neutral.svg';
+import SmileIcon from '~assets/icons/sentiments-icons/smile.svg';
+import SmileDizzyIcon from '~assets/icons/sentiments-icons/smile-dizzy.svg';
+import { ESentiment } from '~src/types';
 
 interface IPostCommentFormProps {
 	className?: string;
-	setCurrentState?:(postId: string, type:string, comment: IComment) => void;
+	isUsedInSuccessModal?: boolean;
+	voteDecision? :string
+	setSuccessModalOpen?: (pre: boolean) => void;
+setCurrentState?:(postId: string, type:string, comment: IComment) => void;
+}
+
+interface IEmojiOption {
+	icon: any;
+	currentSentiment: number;
+	clickable?:boolean;
+	disabled?:boolean;
 }
 
 const commentKey = () => `comment:${global.window.location.href}`;
 
 const PostCommentForm: FC<IPostCommentFormProps> = (props) => {
-	const { className, setCurrentState } = props;
+	const { className , isUsedInSuccessModal = false ,  voteDecision = null, setSuccessModalOpen = () => {return null; },setCurrentState } = props;
```

Comment:
setSuccessModalOpen's default value is an odd function please use `setSuccessModalOpen?.()` instead